res/controllers/Reloop_Beatmix.js

```diff
@@ -0,0 +1,206 @@
+/****************************************************************
+*       Reloop Beatmix script v0.9                              *
+*       Author: Eugene Erokhin, uncle.eugene@gmail.com          *
+*       Tested on Mixxx version 2.4.x                           *
+*                                                               *
+*       Documentation in the Mixxx wiki:                        *
+*       https://mixxx.org/wiki/doku.php/reloop_beatmix          *
+****************************************************************/
+
+var Beatmix = {};
+
+// This constant defines the volume fader threshold at which fader start kicks in.
+const faderStartThr = 0x20;
+
+// Midi notes constants
+const CH1 = 0x90;
+const CH2 = 0x91;
+const FULL = 0x7F;
+const OFF = 0x00;
+
+const bNAVKNOB = 0x18;
+const bBEATMASH1 = 0x21;
+const bBEATMASH2 = 0x22;
+const QUICKJOGR = 0x26;
+const bPLUS = 0x27;
+const QUICKJOGL = 0x28;
+const bMINUS = 0x26;
+const bSHIFTMINUS = 0x46;
+const bSHIFTPLUS = 0x47;
+const bSHIFTCUE = 0x4E;
+const bKEYLOCK = 0x4F;
+const bSHIFTLOAD = 0x52;
```

Comment:
why the `b` prefix and not all capitals for all?

Reply:
Fixed.